templates/themes/peak/pagelayout/header/login.html.twig

```diff
@@ -37,10 +37,16 @@
                 <span class="user-icon user-icon--main"></span>
             {% endif %}
 
-            <a class="btn btn-user" href={{ path('backoffice_dashboard_index') }}>{{ 'layout.my_dashboard'|trans }}</a>
-            <a class="btn btn-user" href={{ path("backoffice_my_account_personal_details") }}>{{ 'layout.my_profile'|trans }}</a>
+            <a class="btn btn-user" href={{ path('backoffice_dashboard_index') }}>{{ 'ngsite.layout.my_dashboard'|trans }}</a>
+            <a class="btn btn-user" href={{ path("backoffice_my_account_personal_details") }}>{{ 'ngsite.layout.my_profile'|trans }}</a>
 
-            <a class="btn btn-user" href="{{ logout_path() }}">{{ 'layout.logout'|trans }}</a>
+            {% if is_granted('ROLE_IMPERSONATED_USER') %}
```

Comment:
I believe this is extra too, since it is added in impersonation PR.

Maybe it's best to keep this open until all other PRs are merged?

Reply:
Maybe @PetraMiseta knows, but yes, we can keep it open

Reply:
Impersonation module is merged, so this should be rebased now.